docs/getting-started/security-ui.asciidoc

```diff
@@ -19,23 +19,47 @@ image::images/search-bar.png[]
 [[page-tabs]]
 == Page tabs
 
-The {security-app} contains seven page tabs that enable analysts to view, analyze, and manage security data:
+The {security-app} contains seven pages that enable analysts to view, analyze, and manage security data:
```

Comment:
There are more than seven pages now; maybe revise to "contains the following pages" or similar?